diesel/src/pg/connection/row.rs

```diff
@@ -43,7 +43,7 @@ impl<'a> PgNamedRow<'a> {
 }
 
 impl<'a> NamedRow<Pg> for PgNamedRow<'a> {
-    fn get_raw_value(&self, index: usize) -> Option<&[u8]> {
+    fn get_raw_value<'b>(&'b self, index: usize) -> Option<PgValue> {
```

Comment:
Either that lifetime is not required or the return value should use a explicit lifetime to make it easier to reason about which lifetime is used there.